test/model.test.js

```diff
@@ -742,6 +742,37 @@ describe.onServer('Remote Methods', function() {
 
       expect(callbackSpy).to.have.been.calledWith(TestModel.sharedClass, 'findOne');
     });
+
+    describe('batch operations', function() {
```

Comment:
I am proposing to move this new test into `describe.onServer('Remote Methods')` block as `describe('Model.create(data, callback')`. I think it would be worth adding two tests - one sending an object, the other sending an array. That way the tests make it clear that both variants are supported.